lib/open_project/translations/models/locales_updater.rb

```diff
@@ -124,4 +128,21 @@ def replace_file(filepath, new_file)
   def js_translation?(translation_file_path)
     !!(translation_file_path.basename.to_s[0..-5] =~ /\Ajs-.+\z/)
   end
+
+  ##
+  # Crowdin has a bug that removes existent translations from new versions
+  # see: https://crowdin.uservoice.com/forums/31787-collaborative-translation-tool/suggestions/8942341-fix-the-recognition-of-duplicated-strings-with-plu
+  def upload_old_translations(plugin_name)
+    @i18n_provider.each_previous_locale do |entry|
+      language_name = entry.name.split('/').first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.